assets/src/edit-story/components/link/guidelines.js

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+import { rgba } from 'polished';
+import { useState, useEffect } from 'react';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { useStory } from '../../app';
+import { useUnits } from '../../units';
+import { CTA_ZONE_PERCENT } from '../../constants';
+import { useTransformHandler } from '../transform';
+import { LinkType, inferLinkType, getLinkFromElement } from '.';
+
+const Separator = styled.div`
+  position: absolute;
+  bottom: ${CTA_ZONE_PERCENT * 100}%;
+  left: 0px;
+  width: 100%;
+  mix-blend-mode: difference;
+  pointer-events: none;
+  z-index: 1;
+  background-image: linear-gradient(
```

Comment:
Why isn't this just a dashed border like on Figma? Seems overly complicated.

Reply:
Sam didn't like it and I agree, the distance between the dashes on the default browser dashed border doesn't look good so I had to improvise

Reply:
Just tested it and have to agree with it 👍